internal/check/handler.go

```diff
@@ -63,6 +63,14 @@ type RESTResponse struct {
 	Allowed bool `json:"allowed"`
 }
 
+// swagger:parameters getCheckRequest
```

Comment:
You have to add the post route here as well for the SDK.
```suggestion
// swagger:parameters getCheckRequest postCheck
```